src/main.py

```diff
@@ -169,8 +169,9 @@ def main():
 
                         continue
 
-                    with open(file_path, "a+", encoding="utf-8") as f:
-                        f.write("\n***\n")
+                    if not args.force:
```

Comment:
Uhhh that's not exactly what I meant

It doesn't seem like expected beavior to have 3 files sent and 2 new ones; user runs with -f and all 5 get sent to anki; then runs normally some other time, after adding maybe 3-4 other files and sees error messages abt duplicates for 2 file's worth of notes.

This has to be in the parse_markdown code or smth -- it should write the delimiter if it's not already there but not write multiple delimiters and that imo should be agnostic of the force flag

Hehe thanks for implementing my nitpicky requests :)

Reply:
ohh ok lol I thought it was probably something more complicated than this.

not sure I follow the force flag scenario, but I'll make a ticket for returning a bool or something from parse_markdown.

also might reimplement this in golang idk hehe